.drone.star

```diff
@@ -928,6 +940,9 @@ def acceptance(ctx):
                         if (params["visualTesting"] or (isLocalBrowser(browser) and params["screenShots"])):
                             steps += githubComment()
 
+                        if (params["earlyFail"]):
+                            steps += stopBuild()
```

Comment:
Also add a GitHub comment step so that we know that the build is being cancelled by some test suite.